src/_data/hint.js

```diff
@@ -1,10 +1,9 @@
 module.exports = {
   en : {
-    heading: "Include hint text and error message text for task success",
+    heading: "Include a hint text for task success",
```

Comment:
This heading is weird to me. Could it be something like "Include hint text for better task success"?

Reply:
Good question, will have to check with Amy since she wrote this.